packages/manager/src/modules/change-password-trading-account/change-password-trading-account-popup.tsx

```diff
@@ -0,0 +1,105 @@
+import { ProgramPwdUpdate } from "gv-api-web";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Dispatch, bindActionCreators } from "redux";
+import Dialog, { IDialogProps } from "shared/components/dialog/dialog";
+import RootState from "shared/reducers/root-reducer";
+
+import ChangePasswordTradingAccountForm, {
+  IChangePasswordTradingAccountFormValues
+} from "./components/change-password-trading-account-form";
+import { changePasswordTradingAccount } from "./services/change-password-trading-account.service";
+
+interface IChangePasswordTradingAccountPopupOwnProps extends IDialogProps {
+  id: string;
+  programName: string;
+}
+
+interface IChangePasswordTradingAccountPopupOtherProps {
```

Comment:
Divide it into two interfaces (`...StateProps` and `...DispatchProps`) and add them to `mapStateToProps` and `mapDispatchToProps`.
shared/components/deposit-details/deposit-details-container.tsx for example.